rpa-manager/modules/ROOT/pages/processautomation-prepare.adoc

```diff
@@ -4,7 +4,7 @@ Create and document the environment for the automation of processes.
 
 == Before You Begin
 
-* Obtain the needed privileges:
+* Ask an administrator to xref:usermanagement-manage.adoc#assign-privileges-to-a-user[assign you the needed privileges:
 +
```

Comment:
For the table, I would unbold the required privileges. There is too much bold, which is distracting. Also, you aren't stepping the user through the GUI here so these don't have to be bold.